src/stuf/stuf_bg/tests/test_client.py

```diff
@@ -185,3 +185,40 @@ def test_errors_are_not_swallowed(self):
 
             with self.assertRaises(Exception):
                 self.client.get_values("999992314", list(FieldChoices.values.keys()))
+
+    def test_inp_heeftAlsKinderen(self):
+        test_bsn = "999992314"
+
+        with STUF_BG_XSD.open("r") as infile:
+            xmlschema_doc = etree.parse(infile)
+        xmlschema = etree.XMLSchema(xmlschema_doc)
+
+        with requests_mock.Mocker() as m:
+            m.post(self.client.service.soap_service.url, status_code=200)
+            self.client.get_values_for_attributes(test_bsn, ["inp.heeftAlsKinderen"])
+
+        request_body = m.last_request.body
+        doc = etree.parse(BytesIO(request_body))
+        soap_body = (
+            doc.getroot()
+            .xpath(
+                "soap:Body",
+                namespaces={"soap": "http://schemas.xmlsoap.org/soap/envelope/"},
+            )[0]
+            .getchildren()[0]
+        )
+
+        # validate soap message with xsd
+        xmlschema.assert_(soap_body)
```

Comment:
This would assert the order of elements is correct, right?

Reply:
I've added the test and reworded things so these things can become apparent from reading the test suite.